### ISSUE TITLE
Application version clash for goldrush

### ISSUE THREAD

**Issue by basus** [FlowForwarding/LINC-Switch#362]
I'm having some trouble building LINC-Switch. It seems like there is something wrong with the `goldrush` dependency:

```
==> rel (generate)
ERROR: generate failed while processing /home/spn/Public/LINC-Switch/rel: {'EXIT',{{badmatch,{error,"goldrush: Application version clash. Multiple directories contain version \"0.1.6\"."}},
         [{rebar_reltool,generate,2,
                         [{file,"src/rebar_reltool.erl"},{line,53}]},
          {rebar_core,run_modules,4,[{file,"src/rebar_core.erl"},{line,434}]},
          {rebar_core,execute,5,[{file,"src/rebar_core.erl"},{line,363}]},
          {rebar_core,process_dir1,6,[{file,"src/rebar_core.erl"},{line,227}]},
          {rebar_core,process_each,5,[{file,"src/rebar_core.erl"},{line,297}]},
          {rebar_core,process_dir1,6,[{file,"src/rebar_core.erl"},{line,203}]},
          {rebar_core,process_commands,2,
                      [{file,"src/rebar_core.erl"},{line,90}]},
          {rebar,main,1,[{file,"src/rebar.erl"},{line,58}]}]}}
make: *** [rel] Error 1
```

When running the Makefile I had to manually change the dependency URL in lager's `rebar.config` file because Github has changed the format of the `git://` URLs.

**Comment by mentels** [comment]
If you still have the problem try to remove the `deps/` directory and run `make rel` again. If that doesn't help please re-open the issue.

**Comment by basus** [comment]
I managed to fix these by using Ubuntu 14.04's Erlang distribution rather than the one from Erlang Solutions.

**Comment by marcsugiyama** [comment]
The Erlang Solutions "Enterprise" distribution includes some open source packages such as goldrush.  I believe this causes the conflict reported in the issue because reltool sees the package in two places, both in the lib directory of the Erlang distribution and in the deps directory.  The solution is to remove one of them. As already noted, using a standard Erlang/OTP distribution, rather than the "Enterprise" distribution from Erlang Solutions, is one way of doing that.
